app/client/components/history/historyDrawer.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Drawer, Timeline } from "antd";
+import { connect } from "react-redux";
+import { toTitleCase, replaceUnderscores } from "../../utils";
+interface Props {
+  drawerVisible: boolean;
+  onClose: any;
+  changes: [];
+}
+
+class HistoryDrawer extends React.Component<Props> {
+  constructor(props: any) {
+    super(props);
+  }
+
+  render() {
+    const stringList = this.props.changes.map(change => {
+      return `${toTitleCase(
+        replaceUnderscores(change.column_name)
+      )} changed from "${change.old_value}" to "${change.new_value}" by ${
+        change.change_by.email
+      }`;
+    });
```

Comment:
Can we have the email in a teal color?

How hard is it to display the DateTime for each item?

Reply:
Done!